LibreHardwareMonitorLib/Hardware/Cpu/Amd10Cpu.cs

```diff
@@ -125,11 +125,6 @@ public Amd10Cpu(int processorIndex, CpuId[][] cpuId, ISettings settings) : base(
                     }
                     break;
                 }
-                case 0x17:
-                {
-                    miscellaneousControlDeviceId = FAMILY_17H_MODEL_00_MISC_CONTROL_DEVICE_ID;
-                    break;
-                }
```

Comment:
I don't think this file should be modified?

Reply:
This make no sense in the AMD10 class.

Reply:
Ah, right, I thought it was the Ryzen class.